django/views/csrf.py

```diff
@@ -1,10 +1,9 @@
 from django.conf import settings
 from django.http import HttpResponseForbidden
-from django.template import Context, Engine
+from django.template import Context, Engine, TemplateDoesNotExist, loader
 from django.utils.translation import ugettext as _
 from django.utils.version import get_docs_version
 
-
```

Comment:
No need to push a change for this and trigger a CI build, just leaving a note to not commit this line change.